internal/driver/commands.go

```diff
@@ -131,6 +131,12 @@ var pprofVariables = variables{
 	// Filename for file-based output formats, stdout by default.
 	"output": &variable{stringKind, "", "", helpText("Output filename for file-based outputs")},
 
+	"comment": &variable{stringKind, "", "", helpText(
+		"Comment string to add to the profile",
+		"The string is appended to any output format that includes comments,",
```

Comment:
The long help message is a bit confusing. How about something like:

"This adds free-form annotations to the profile, which are generated on some reports or can be explicitly queries with pprof -comments"

Reply:
Done.